vochain/scrutinizer/vote.go

```diff
@@ -27,6 +27,14 @@ var ErrNoResultsYet = fmt.Errorf("no results yet")
 // implementation, thus let badgerhold abstract it.
 const kvErrorStringForRetry = "Transaction Conflict"
 
+// GetVoteReference gets the reference for an AddVote transaction.
+// This reference can then be used to fetch the vote transaction directly from the BlockStore.
+func (s *Scrutinizer) GetVoteReference(nullifier []byte) (*VoteReference, error) {
+	txRef := &VoteReference{}
+	err := s.db.FindOne(txRef, badgerhold.Where(badgerhold.Key).Eq(nullifier))
+	return txRef, err
```

Comment:
You can do: `return txRef, s.db.FindOne(txRef, badgerhold.Where(badgerhold.Key).Eq(nullifier))`